docs/assets/diagrams/transfer/03a-confirm-payee.puml

```diff
@@ -0,0 +1,127 @@
+@startuml
+
+title Confirm Payee
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+Alice -> App: The payee info is correct
+
+rnote right of App #Light
+  ""updateDoc(""
+  ""  "transactions/users/1234/<uuid>",""
+  ""  {""
+  ""    sourceAccountId: "bob.fspa",""
+  ""    consentId: 123,""
+  ""    amount: {""
+  ""      amount: 20.00""
+  ""      currency: "USD"""
+  ""    },""
+  ""    status: "PAYEE_CONFIRMED"""
+  ""  }""
+  "")""
+end rnote
+
+App -> Firebase: Update transaction status
+return ""200 OK""
+
+rnote right of Firebase #Light
+  ""event(""
+  ""  "document.update",""
+  ""  "transactions/users/1234/<uuid>",""
+  ""  {""
+  ""    sourceAccountId: "bob.fspa",""
+  ""    consentId: 123,""
+  ""    amount: {""
+  ""      amount: 20.00""
+  ""      currency: "USD"""
+  ""    },""
+  ""    status: "PAYEE_CONFIRMED"""
+  ""  }""
+  "")""
+end rnote
+
+Firebase -> Server ++: Notify update on transaction status
+Firebase <-- Server: ""200 OK""
+
+rnote right of Server #Light 
+  ""**POST /thirdpartyRequests/transactions**""
+  ""FSPIOP-Source: ...""
+  ""FSPIOP-Destination: ...""
+  ""{""
+  ""  "transactionRequestId": string,""
+  ""  "sourceAccountId": string,""
```

Comment:
Is every value here empty?

Reply:
Nope, just updated with the example values...

However, for some other fields, I currently still use placeholders as I'm not quite sure about the exact details and perhaps it may not be a big concern for the sequence diagram?

Reply:
All updated with examples